### ISSUE TITLE
Please remove the `postinstall` hook

### ISSUE THREAD

**Issue by silverwind** [react-hookz/deep-equal#269]
I don't have `husky` on my system, and this seems like an oversight:

```
$ npm i -D @react-hookz/deep-equal
npm error code 127
npm error command failed
npm error command sh -c husky
npm error sh: husky: command not found
npm error Log files were not written due to the config logs-max=0
```

**Comment by xobotyi** [comment]
I don't really understand how is it working on you, since in published `package.json` postinstall script being renamed to `_postinstall` and therefore cant be invoked...
@silverwind could you please provide a `package.json` contents of package you get?

**Comment by silverwind** [comment]
It's odd, the script is prefixed with underscore in package.json but npm still seems to identify it as a postinstall script:

```
$ npm --version
10.9.2
$ npm i --ignore-scripts @react-hookz/deep-equal@3.0.1
$ cat node_modules/@react-hookz/deep-equal/package.json | jq .scripts
{
  "_postinstall": "husky",
  "prepack": "pinst --disable",
  "postpack": "pinst --enable",
  "build": "yarn build:clean && tsc --version && tsc -p ./tsconfig.build.json",
  "build:clean": "rimraf -g ./dist",
  "lint": "eslint",
  "lint:fix": "eslint --fix",
  "test": "vitest",
  "test:coverage": "vitest --coverage",
  "benchmark": "vitest bench --run"
}
$ npm view @react-hookz/deep-equal@3.0.1 scripts
{
  postinstall: 'husky',
  prepack: 'pinst --disable',
  postpack: 'pinst --enable',
  build: 'yarn build:clean && tsc --version && tsc -p ./tsconfig.build.json',
  'build:clean': 'rimraf -g ./dist',
  lint: 'eslint',
  'lint:fix': 'eslint --fix',
  test: 'vitest',
  'test:coverage': 'vitest --coverage',
  benchmark: 'vitest bench --run'
}
```

I guess you should remove it completely before publish to be sure.

https://github.com/npm/cli/issues/7322 sounds related.

**Comment by silverwind** [comment]
According to https://github.com/npm/cli/issues/7322#issuecomment-2242926671, npm >= 10.4 will obtain package metadata like `scripts` from the npm registry, not package.json. So for some reason, the data on the registry does not match what's in package.json for this package:

```
$ curl -s 'https://registry.npmjs.org/@react-hookz/deep-equal' | jq '.versions["3.0.1"].scripts'
{
  "postinstall": "husky",
  "prepack": "pinst --disable",
  "postpack": "pinst --enable",
  "build": "yarn build:clean && tsc --version && tsc -p ./tsconfig.build.json",
  "build:clean": "rimraf -g ./dist",
  "lint": "eslint",
  "lint:fix": "eslint --fix",
  "test": "vitest",
  "test:coverage": "vitest --coverage",
  "benchmark": "vitest bench --run"
}
```

**Comment by xobotyi** [comment]
fixed with d716a150841a6000c0eb497cefb6a028d24dbfda and release v3.0.3

```shell
> npm view @react-hookz/deep-equal@3.0.3 scripts 
{
  _postinstall: 'husky',
  build: 'yarn build:clean && tsc --version && tsc -p ./tsconfig.build.json',
  'build:clean': 'rimraf -g ./dist',
  lint: 'eslint',
  'lint:fix': 'eslint --fix',
  test: 'vitest',
  'test:coverage': 'vitest --coverage',
  benchmark: 'vitest bench --run'
}
```

**Comment by silverwind** [comment]
Thanks
